### ISSUE TITLE
dsitinguish between shell features and external commands

### ISSUE THREAD

**Issue by GReagle** [clarkgrubb/hyperpolyglot#42]
I think it would be great if this document were to distinguish between shell features and external commands, by using a special color or font, for example, for external commands, in the tables.

For example, "simple command with environment variable" in table Grammar.  Some of the shells do not support this feature themselves, but the env command can be used for the purpose.  But env is an external command, so it doesn't work with shell functions or aliases.  So it would be really convenient to see at a glance (e.g. color or font style or whatever) that env is an external command.

**Comment by limeblack** [comment]
@GReagle I don't think this going to happen because I believe a Wiki is being used to generate the site which means you really can't support special formatting.

**Comment by GReagle** [comment]
The wikis that I am familiar with are capable of doing such things as **bold** and *italic*.
